tasmota/xnrg_12_solaxX1.ino

```diff
@@ -492,21 +579,26 @@ bool Xnrg12(uint8_t function)
 
   switch (function) {
     case FUNC_EVERY_250_MSECOND:
-      solaxX1250MSecond();
+      solaxX1_250MSecond();
       break;
     case FUNC_JSON_APPEND:
-      solaxX1Show(1);
+      solaxX1_Show(1);
       break;
 #ifdef USE_WEBSERVER
     case FUNC_WEB_SENSOR:
-      solaxX1Show(0);
+      solaxX1_Show(0);
       break;
 #endif  // USE_WEBSERVER
     case FUNC_INIT:
-      solaxX1SnsInit();
+      solaxX1_SnsInit();
       break;
     case FUNC_PRE_INIT:
-      solaxX1DrvInit();
+      solaxX1_DrvInit();
+      break;
+    case FUNC_COMMAND:
+      if (XNRG_12 == XdrvMailbox.index) { // "EnergyConfig12 <command>"

```

Comment:
No need to check for XNRG_12 as these functions are only called if XNRG_12 is the active energy service driver. There can be only ONE active energy service driver.